var/spack/repos/builtin/packages/wi4mpi/package.py

```diff
@@ -24,7 +25,7 @@ class Wi4mpi(CMakePackage):
             description='The build type to build',
             values=('Debug', 'Release', 'RelWithDebInfo'))
 
-    depends_on('mpi')
+    depends_on('openmpi')
```

Comment:
Why this change? Does this package depend on OpenMPI internals?

Reply:
Closing, since looking at this https://github.com/cea-hpc/wi4mpi it seems we are modeling the package wrongly with an hard-coded dependency on `openmpi`. Feel free to reopen if you want to continue the work here.